patches/app-builder-lib+24.13.3.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/node_modules/app-builder-lib/templates/msi/template.xml b/node_modules/app-builder-lib/templates/msi/template.xml
+index 2d5cd3c..04ed672 100644
+--- a/node_modules/app-builder-lib/templates/msi/template.xml
++++ b/node_modules/app-builder-lib/templates/msi/template.xml
+@@ -26,6 +26,11 @@
+     <Property Id="ARPPRODUCTICON" Value="${iconId}"/>
+     {{ } -}}
+ 
++    <CustomAction Id="removeExeInstaller" Directory="APPLICATIONFOLDER" ExeCommand="[APPLICATIONFOLDER]Uninstall ${productName}.exe /S" Execute="immediate" Impersonate="yes" Return="asyncWait"/>
```

Comment:
This patch adds a step to the MSI to check for per-user installation of the NSIS installer. If that installation exists, it will run the uninstall program. This covers the case where a user might have the NSIS build installed from a previous build, or from auto-update, to avoid duplicate installations on the system.

@jupenur I called this one out in our security meeting, because I'm not sure how comfortable we are executing an arbitrary EXE like this. Let me know your thoughts on this one.

Reply:
This seems safe since it's an absolute path to a directory where you wouldn't be able to drop files e.g. from a web browser.